scripts/gh-pr-assigned.coffee

```diff
@@ -18,11 +21,24 @@
 #
 # Authors:
 #   ram535ii
+#   Thanks to https://github.com/hubot-scripts/hubot-github-repo-event-notifier for inspiration
+
+room = process.env["HUBOT_GITHUB_EVENT_NOTIFIER_ROOM"]
+user = process.env["HUBOT_GITHUB_USERNAME"]
+action = "assigned"
+
+if !room?
+  console.warn "A room must be specific for posting to"
+
+if !user?
+  console.warn "A user to check for pull request assignment must be specified"
+
 module.exports = (robot) ->
   robot.router.post "/hubot/gh-repo-events", (req, res) ->
     data = req.body
-    if data.action == "assigned" && data.assignee.login == "ram535ii"
-      console.log "Pull request assigned to constantijn"
-      robot.messageRoom "constantijnschepens", "ASSIGNED TO A NEW PR BIATCH - check it here => " + req.body.pull_request.html_url
+    pr_url = data.pull_request.html_url
+
+    if data.action == action && data.assignee.login == user
```

Comment:
Howcome you abstracted this to `action`? The whole condition is all about assignation so "assigned" seemed sensible to me.